calico_containers/tests/st/utils.py

```diff
@@ -0,0 +1,28 @@
+import sh
+from sh import docker
+
+
+def get_ip():
+    """Return a string of the IP of the hosts eth0 interface."""
+    intf = sh.ifconfig.eth0()
+    return sh.perl(intf, "-ne", 's/dr:(\S+)/print $1/e')
```

Comment:
I'd prefer Rob's suggestion here.  I'd rather depend on internet access than having an interface named eth0, which we know isn't the case for RedHat and other popular distros.